src/main/java/org/openhds/service/impl/census/IndividualService.java

```diff
@@ -39,9 +71,45 @@ public Individual recordIndividual(Individual individual,
                                        String membershipUuid,
                                        String relationshipUuid,
                                        String fieldWorkerId) {
+
         individual.setCollectedBy(fieldWorkerService.findOrMakePlaceHolder(fieldWorkerId));
 
-        //TODO: Handle side effect creation for things like socialgroup and membership etc.
+        //Create the individual first, so that everything after is guaranteed to point at a real entity
+        individual = createOrUpdate(individual);
+
+        //TODO: Is the socialgroup created here?
+        //TODO: Do we make two paths depending on if it's a member or head of household?
+        SocialGroup socialGroup = socialGroupService.findOrMakePlaceHolder(socialGroupUuid);
+        Location location = locationService.findOrMakePlaceHolder(locationUuid);
+
+        //TODO: How are we going to start using project codes?
+        //TODO: Do these side effect entities need to call findOrMakePlaceholder?
```

Comment:
I think we should always create entities with findOrMakePlaceholder.  Otherwise we will have to impose ordering on when things are created.

Any entity can be POSTED or created at any time.  So what happens if a Residency update or correction arrives before the individual?  findOrMakePlaceholder will find it and preserve the references we want.  If we create the residency directly, we would end up with an extra residency floating around.